samples/cellular/modem_shell/src/sock/sock_shell.c

```diff
@@ -429,11 +429,12 @@ static int cmd_sock_connect(const struct shell *shell, size_t argc, char **argv)
 			arg_secure = true;
 			break;
 		case 'T': /* Security tag */
-			arg_sec_tag = atoi(optarg);
-			if (arg_sec_tag < 0) {
+			err = 0;
+			arg_sec_tag = shell_strtoul(optarg, 10, &err);
+			if (err) {
 				mosh_error(
-					"Valid range for security tag (%d) is 0 ... 2147483647.",
-					arg_sec_tag);
+					"Valid range for security tag (%s) is 0 .. %u.",
+					optarg, ULONG_MAX);
```

Comment:
I think that `UINT32_MAX` fits better and is more readable.
```suggestion
					optarg, UINT32_MAX);
```

Reply:
Ok